eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -0,0 +1,250 @@
+parameters:
+  Artifacts: []
+  ArtifactName: 'not-specified'
+
+stages:
+  - stage: Signing
+    dependsOn: ${{parameters.DependsOn}}
+    jobs:
+      - deployment: SignPackage
+        environment: esrp
+        timeoutInMinutes: 20
+        pool:
+          vmImage: vs2017-win2016
```

Comment:
This should be updated to the latest win image.

Reply:
windows-2019

Reply:
fixed